### ISSUE TITLE
Add a miscellaneous modifier to individual skills?

### ISSUE THREAD

**Issue by mraichelson** [mraichelson/cyberpunk-red-tabbed#14]
For handling things like Tech Hair, Chem Skin, or Agent bonuses.

**Comment by TekDragon** [comment]
Yes, please. I don't know why folks aren't putting this at the top. There are SO MANY skills and SO MANY items/cyberware that provide bonuses to skills. Between an agent, techscanner, lockpicking set, and some cyberware - my tech has bonuses to about a dozen skills. As is, I have to write all those down elsewhere, then manually add them to the Custom field in the Modifer section of the Skills page. For each freaking roll. 

Alternatively, I can add those modifiers directly to the base skill, meaning I need to write those values somewhere else so I don't lose track as gear is replaced and skills increased by XP. 

Both those solutions suck. Please address this. It doesn't need to be crazy complex. Just make it so skills have 3 columns. One for your base skill level at creation/level-up, one for modifiers, then a total (that automatically adds in your stat points and, optionally, your automatic points to certain skills at creation).
